content/management/configuration-api/v3.3/index.mdx

```diff
@@ -3558,14 +3559,20 @@ curl -X POST \
 
 <Text>
 
-### List Registered Webhooks
+### List Webhooks
```

Comment:
Lists all webhooks registered for the **Client Id** associated with the access token passed in the request. If you authorize with a PAT, you need to specify the Client Id.

Reply:
passed, used, submitted?